src/applications/hca/helpers.jsx

```diff
@@ -6,15 +6,19 @@ import AdditionalInfo from '@department-of-veterans-affairs/formation/Additional
 import {
   stringifyFormReplacer,
   filterViewFields,
-  filterInactivePages,
```

Comment:
This was removed in the new branch we're using.

Reply:
If this is a common use case, would it make sense to create a `filterInactivePages` platform util to abstract this logic?

Reply:
So far we've used it in one form, so I would say it's not common.